scene/gui/text_edit.cpp

```diff
@@ -3370,6 +3370,21 @@ void TextEdit::cut() {
 	}
 }
 
+void TextEdit::convert_case(const String &p_Case) {
+	if (selection.active) {
+		String text = _base_get_text(selection.from_line, selection.from_column, selection.to_line, selection.to_column);
+		selection.active = false;
+		_remove_text(selection.from_line, selection.from_column, selection.to_line, selection.to_column);
+		cursor_set_line(selection.from_line);
+		cursor_set_column(selection.from_column);
+		if (p_Case == "UPPER") {
```

Comment:
If you want to do it this way, it should be an enum, not a magic string.

Reply:
:p